public/index.html

```diff
@@ -28,7 +50,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>MA Covid Vaccines</title>
+    <title>Covid-19 Vaccine Appointments in Massachusetts</title>
```

Comment:
The longest I'm willing to make this is "Massachusetts COVID Vaccines" - we don't want this too long for usability purposes

Reply:
Why?  When is this title used aside from SEO?  When do people see it aside from a truncated tab which is why we moved the key words up front.

Reply:
![image](https://user-images.githubusercontent.com/1749962/108636687-e3a13580-7454-11eb-97b7-b8d484c3f901.png)

This is what it looks like in Chrome for example.

Reply:
Alright, I'm good given your argument on moving key words up front! Approving.